articles/iot-edge/how-to-connect-downstream-device.md

```diff
@@ -35,7 +35,10 @@ Before following the steps in this article, you should have two devices ready to
 2. A downstream device that has a device identity from IoT Hub. 
     You cannot use an IoT Edge device as the downstream device. Instead, use a device registered as a regular IoT device in IoT Hub. In the portal, you can register a new device in the **IoT devices** section. Or you can use the Azure CLI to [register a device](../iot-hub/quickstart-send-telemetry-c.md#register-a-device). Copy the connection string and have it available to use in later sections. 
 
-    Currently, only downstream devices with symmetric key authentication can connect through IoT Edge gateways. X.509 certificate authorities and X.509 self-signed certificates are not currently supported. 
+    Currently, only downstream devices with symmetric key authentication can connect through IoT Edge gateways. X.509 certificate authorities and X.509 self-signed certificates are not currently supported.
```

Comment:
This is actually no longer true. Version 1.0.6 of the Edge Hub supports downstream devices connecting to the Edge Hub using a x.509 certificate if the downstream device is in the Edge Device's scope (as part of the extended offline feature).